build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+  id "io.spring.dependency-management" version "1.0.5.RELEASE" apply false
+}
+
+allprojects {
+  group 'io.knotx'
+
+  repositories {
+    jcenter()
+    maven { url "https://plugins.gradle.org/m2/" }
+    maven { url "https://oss.sonatype.org/content/groups/staging/" }
+  }
+}
+
+description = 'Knot.x JUnit5 test module'
+
+defaultTasks = ['build']
+
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'io.spring.dependency-management'
+
+dependencyManagement {
+  imports {
+    mavenBom 'io.knotx:knotx-dependencies:1.3.1-SNAPSHOT'
+    mavenBom 'org.junit:junit-bom:5.2.0'
+  }
+}
+
+tasks.withType(JavaCompile) {
+  options.encoding = 'UTF-8'
+  options.compilerArgs << '-Xlint:deprecation'
+  options.incremental = true
+}
+
+
+dependencies {
+  compile 'io.knotx:knotx-core'
```

Comment:
Why does knotx-junit5 depend on Knot.x Core. I think that the dependency should be: Knot.x Core depends on knotx-junit5.

Reply:
Due to a requirement to mock `io.knotx.dataobjects.Fragment` via Mockito in `KnotxArgumentConverter` class. I don't currently have a sane workaround for this.